### PR TITLE
A simple ToDo program.

### PR DESCRIPTION
**Pull request by EmekaEmmanuel**
This project is a simple program to track our To Do tasks; Built using webpack and served by a webpack dev server.

### CODE REVIEW

**Review by jasonfelice — COMMENTED** [review]
Hi @EmekaEmmanuel, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

- Please make sure all the linter tests are passing before submitting another request. 😄 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/index.html [hunk 1]
```diff
@@ -17,7 +17,6 @@
         
     </section>
 
-</body>
-<!-- <script type="module" src="./index.js"></script> -->
```
Comment (Gambit142) [inline src/index.html:21]: Although I am highlighting this piece of code, the changes I am about to request are not related to it. I could not highlight the relevant piece of code in the Files Changes section of your pull request. Kindly make the following changes
- [ ] Kindly remove the Microverse logo from your README.md file. If you have your logo, you can use it but if you don't, kindly remove it. 👍 💻 
- [ ] Kindly remove appreciation for Cindy Shin in the Acknowledgement Section of your README.md file. I guess that was used during the HTML/CSS/JavaScript capstone project. 👍 😄 
- [ ] Kindly remove all unused/commented codes (Line 7, Line 16, Line 43 - 46) in your _**webapack.config.js**_ file. Removing such makes your code look clean.
- [ ]  Kindly remove the unused/commented code (Line 24) in your _**taskData.js**_ file. Removing such makes your code look clean.

**Review by Gambit142 — CHANGES_REQUESTED** [review]
Hi @EmekaEmmanuel 👋, 

Good job so far!

## Highlights
- All linters checks passed ✔️
- Webpack was used to bundle JavaScript ✔️
- Tasks are rendered dynamically to the DOM ✔️
- CSS is loaded by Webpack Style/CSS Loader ✔️ 

There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Kindly check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:

N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### Please, remember to tag me in your question so I can receive the notification. You can tag me by typing @Gambit142

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by Ismailco — COMMENTED** [review]
Hi @EmekaEmmanuel, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has some linter errors, kindly make sure to fix them before resubmitting

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: README.md [hunk 2]
```diff
@@ -3,7 +3,7 @@
 <a name="readme-top"></a>
 
 <div align="center"> 
-  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <img src="" alt="logo" width="140"  height="auto" />
```
Comment (sja-thedude) [inline README.md]: - [ ] Kindly remove the microverse logo as suggested. Currently its a broken image so remove it.

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Required Changes ♻️

Hi @EmekaEmmanuel, This is @sja-thedude your code reviewer.

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

200w

 ## Highlights
✔️ All linters passing
✔️ Correct Github flow

 ## Required Changes ♻️
- [ ] _Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by danmainah — APPROVED** [review]
Hi @EmekaEmmanuel , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 

## Highlights
✔️ No linter errors
✔️ Github flow was correctly used

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**